.gitlab-ci.yml

```diff
@@ -1,8 +1,3 @@
-test-3.5:
```

Comment:
```
Requirements
Python 3.5 and higher is required.
```

doest it mean, that we won't support py 3.5? if yes, let's remove info from readme

Reply:
Yes, we won't. There is `Python 3.6 and higher is required.` already.